src/common/MPPProber.cc

```diff
@@ -55,7 +55,14 @@ void MPPProber::run()
             });
         }
 
-        scan();
+        try
+        {
+            scan();
```

Comment:
This may also affects the LTS branch. If the sub-thread throw exception without being catched, tiflash may crash. We need to cherry-pick the fix to LTS branch

Reply:
It seems that `scan` does not throw an exception. @guo-shaoge thought that the grpc call would throw an exception, but it actually would not. Adding a try-catch here is fine, but there is no need to do a check-pick.

Reply:
OK